package.json

```diff
@@ -10,7 +10,8 @@
   "scripts": {
     "start": "ember server",
     "build": "ember build",
-    "test": "ember test"
+    "test": "ember test",
+    "seed": "node seed.js"
```

Comment:
I'm sure that there is a better way of doing this, but I don't know how to mount stuff like that into ember.. Perhaps there is some kind of addon?